### ISSUE TITLE
[BUG] Autoconf exited with error

### ISSUE THREAD

**Issue by derdek** [bunkerity/bunkerized-nginx#164]
**Description**
root@name:~# docker run -v /var/run/docker.sock:/var/run/docker.sock:ro -v bunkerized-nginx-configs:/etc/nginx bunkerity/bunkerized-nginx-autoconf
```
[*] Starting autoconf ...
[2021-07-21 18:38:23] [*] Generating config for subdomain.example.com ...
[2021-07-21 18:38:23] [*] Generator output :
[2021-07-21 18:38:23] [*] Generation done !

[2021-07-21 18:38:23] [*] Generated config for subdomain.example.com
[2021-07-21 18:38:23] [*] Listening for Docker events ...
[2021-07-21 18:39:04] [*] Activating config for subdomain.example.com ...
[2021-07-21 18:39:04] [*] Activated config for subdomain.example.com
[2021-07-21 18:39:05] [*] Deactivating config for subdomain.example.com
Traceback (most recent call last):
  File "/opt/entrypoint/app.py", line 67, in <module>
    autoconf.process(server, event["Action"])
  File "/opt/entrypoint/AutoConf.py", line 82, in process
    self.__process_server(obj, event, id, name, labels)
  File "/opt/entrypoint/AutoConf.py", line 176, in __process_server
    if self.__config.reload() :
TypeError: reload() missing 1 required positional argument: 'instances'
[*] autoconf stopped
```

**Comment by fl0ppy-d1sk** [comment]
It should be fixed in the new v1.2.8 version.

**Comment by fl0ppy-d1sk** [comment]
Closing this as it should be resolved.
